tests/integration/test_active_response/test_execd/test_execd_restart.py

```diff
@@ -20,9 +15,8 @@
 from wazuh_testing.tools.monitoring import FileMonitor
 from wazuh_testing.tools.authd_sim import AuthdSimulator
 from wazuh_testing.tools.remoted_sim import RemotedSimulator
-from conftest import AR_LOG_FILE_PATH, set_ar_conf_mode, set_debug_mode, \
-    wait_received_message_line, wait_start_message_line, \
-    wait_ended_message_line, test_version, start_log_monitoring
+from wazuh_testing.execd import AR_LOG_FILE_PATH, wait_received_message_line, \
+    wait_start_message_line, wait_ended_message_line, start_log_monitoring
```

Comment:
Same as before. This improves the readability of the script.

Reply:
https://github.com/wazuh/wazuh-qa/pull/1235/commits/129ec4dd555dc61dc83723849962dc8dd861ecbb